http/src/main/resources/reference.conf

```diff
@@ -96,7 +94,7 @@ gce {
   runtimeDefaults {
     machineType = "n1-standard-4"
     diskSize = 30 # This is default size for just user disk
-    bootDiskSize = 60
+    bootDiskSize = 70
```

Comment:
I increased this a bit more since we now cache all the images vs previously we only cache 1 `base-cu110` based image